packages/react-tokens/scripts/writeTokens.js

```diff
@@ -72,14 +72,15 @@ function writeTokens(tokens) {
 
     // Legacy token support -- values may be incorrect.
     Object.values(tokenValue)
-      .map(values => Object.entries(values))
+      .map((values) => Object.entries(values))
       .reduce((acc, val) => acc.concat(val), []) // flatten
-      .forEach(([oldTokenName, { name, value }]) => {
+      .forEach(([oldTokenName, { name, value, darkThemeValue }]) => {
         const isChart = oldTokenName.includes('chart');
         const oldToken = {
           name,
           value: isChart && !isNaN(+value) ? +value : value,
-          var: isChart ? `var(${name}, ${value})` : `var(${name})` // Include fallback value for chart vars
+          var: isChart ? `var(${name}, ${value})` : `var(${name})`, // Include fallback value for chart vars
+          darkThemeValue: isChart && !isNaN(+darkThemeValue) ? +darkThemeValue : darkThemeValue
```

Comment:
The charts wouldn't use `darkThemeValue` at all. The charts use the fallback value syntax on line 82.

Reply:
Can you explain what you mean?
Anyway I put the failsafe there just in case, from my checks there aren't actually any chart values with dark theme value

Reply:
You added a `isChart` test on line 83, so I assume this is intended to be used with charts?

Reply:
As `darkThemeValue` is similar to `value`, I used the same logic, I can remove it if its not needed?